packages/core/cli.ts

```diff
@@ -190,6 +190,24 @@ yargs(process.argv.slice(2))
 					type: "number",
 					group: "Fuzzer:",
 					default: 5000,
+				})
+				.array("excludeBugDetectors")
```

Comment:
There is already the `custom_hooks` parameter to allow users to write their own hooks / bug detectors. Shouldn't this be called something similar like `excludedHooks` (Jazzer calls it `disabledHooks`)? As discussed, IMHO distinguishing between hooks and bug detect complicates things unnecessary.

Reply:
will proceed as discussed